ui/css/icons.styles.js

```diff
@@ -0,0 +1,21 @@
+// icon styles, setting general height and width properties to solves scaling and consistency problems
```

Comment:
@Abhishek-kumar09 Did you also test this on browser zoom in and out on Safari? Are the icons scaling? With the defined dimensions?

Reply:
Yes, I did it.